### ISSUE TITLE
Trying to update, can't find original git clone folder

### ISSUE THREAD

**Issue by atuttle** [cflint/CFLint#150]
I'm currently on an older version:

```bash
$ cflint --version
CFLint 0.5.0
```

Unfortunately I have no idea where I cloned this git repo into originally and can't find it. I cloned it again and ran `mvn clean install`, which says it completed successfully:

```
[INFO] Installing /Users/atcodes/DEV/CFLint/target/CFLint-0.6.1.jar to /Users/atcodes/.m2/repository/com/github/cflint/CFLint/0.6.1/CFLint-0.6.1.jar
[INFO] Installing /Users/atcodes/DEV/CFLint/pom.xml to /Users/atcodes/.m2/repository/com/github/cflint/CFLint/0.6.1/CFLint-0.6.1.pom
[INFO] Installing /Users/atcodes/DEV/CFLint/target/CFLint-0.6.1-all.jar to /Users/atcodes/.m2/repository/com/github/cflint/CFLint/0.6.1/CFLint-0.6.1-all.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:05 min
[INFO] Finished at: 2016-04-07T08:52:04-04:00
[INFO] Final Memory: 27M/419M
[INFO] ------------------------------------------------------------------------
```

But this doesn't seem to have upgraded my global install:

```bash
$ cflint --version
CFLint 0.5.0
```

`which cflint` points me to `usr/bin/cflint` which is a symlink to `/usr/local/lib/cflint/bin/cflint`, but that's an executable shell script and from there I'm lost.

How to I get my newly built version of cflint to replace the old one?

**Comment by mjclemente** [comment]
When I installed it, I had to point the `cflint` symlink to the binary it generated here: `/installationPath/CFLint/target/appassembler/bin/cflint`.

**Comment by atuttle** [comment]
With the default `mvn clean install` the only thing inside appassembler for me is an `ant` folder containing a `build.xml` file.

I found extra arguments (which looked familiar; I must have used them the first time around) in [build-instructions.md](https://github.com/cflint/CFLint/blob/master/build-instructions.md)... Changed them to: `mvn clean install -D assembleDirectory=target/appassembler` and everything from `/usr/local/cflint` is now in my target/appassembler folder. Running that executable with `--version` does indeed indicate it's 0.6.1 (current latest)... so now I just need to move those files into place or change the symlink.
